File: redsheet-portal/src/app/database-models/v-our-team-member-record.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vOurTeamMemberRecord {
    constructor(vOurTeamMemberRecord?: vOurTeamMemberRecord )  {
        let properties = [ "agreeableId", "assertivenessId", "conflictStyleId", "consciousnessId", "emotionalCompetenceId", "openMindedId", "ourTeamMemberId", "outingId", "personalCalmId", "solutionFocusedId", "teamRoleId", "userId", "willToWinId",];
        for (let property of properties)
            if (vOurTeamMemberRecord && vOurTeamMemberRecord[property])
                this[property] = vOurTeamMemberRecord[property];
    }
 
    @range(1,2147483647)
	agreeableId : number =   undefined;
 
    @range(1,2147483647)
	assertivenessId : number =   undefined;
 
    @range(1,2147483647)
	conflictStyleId : number =   undefined;
 
    @range(1,2147483647)
	consciousnessId : number =   undefined;
 
    @range(1,2147483647)
	emotionalCompetenceId : number =   undefined;
 
    @range(1,2147483647)
	openMindedId : number =   undefined;
 
	ourTeamMemberId : number =   0 ;
 
    @range(1,2147483647)
	outingId : number =   undefined;
 
    @range(1,2147483647)
	personalCalmId : number =   undefined;
 
    @range(1,2147483647)
	solutionFocusedId : number =   undefined;
 
    @range(1,2147483647)
	teamRoleId : number =   undefined;
 
    @range(1,2147483647)
	userId : number =   undefined;
 
    @range(1,2147483647)
	willToWinId : number =   undefined;


}

File: RedSheet.DbEntities/Models/vOurTeamMemberRecord.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vOurTeamMemberRecords", Schema = "dbo")]
	[Serializable]
	public partial class vOurTeamMemberRecord  
	{
 
	[Range(1, int.MaxValue)]
	public int AgreeableId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int AssertivenessId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ConflictStyleId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int ConsciousnessId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int EmotionalCompetenceId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int OpenMindedId { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int OurTeamMemberId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int OutingId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int PersonalCalmId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int SolutionFocusedId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int TeamRoleId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int UserId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int WillToWinId { get; set; } 

	public vOurTeamMemberRecord()
	{
    }

    }
}

